static/about.html

```diff
@@ -72,201 +72,209 @@ <h1 class="text-center" style="color: white;">What is a Polygenic Risk Score?</h
-              with similar risk scores for many traits eliminates potential confounding factors between these
-              traits.<br>
-            </p>
-          </div>
-          <li class="listItems" style="font-size: 1.5em" onclick="displayPanel('filterPanel')">
-            <i id="filterPanel_sign" class="fa fa-angle-down" aria-hidden="true" style="margin-right:5px;"></i>
-            <a>Additional Filtering in Genome-wide Association Studies</a>
-          </li>
-          <div id="filterPanel"
-            style="height: 0px; display: block; overflow: hidden; transition: height 0.5s; -webkit-transition: height 0.5s;">
-            <p>Polygenic risk scores can act as an additional filter for novel loci associated with a trait. If an
-              indiviudal has a trait, but has a low risk score, or does not have a trait, but has a high risk score,
-              the individual likely has novel genetic polymorphisms that need futher study.<br>
-            </p>
-          </div>
-          <li class="listItems" style="font-size: 1.5em" onclick="displayPanel('mdsPanel')">
-            <i id="mdsPanel_sign" class="fa fa-angle-down" aria-hidden="true" style="margin-right:5px;"></i>
-            <a>Mendelian Randomization Studies</a>
-          </li>
-          <div id="mdsPanel"
-            style="height: 0px; display: block; overflow: hidden; transition: height 0.5s; -webkit-transition: height 0.5s;">
-            <p>Mendelian randomization studies use polygenic risk scores to predict the effect a particular
-              treatment or exposure will have on an indiviual.<br>
-            </p>
-          </div>
-          <li class="listItems" style="font-size: 1.5em" onclick="displayPanel('inferPanel')">
-            <i id="inferPanel_sign" class="fa fa-angle-down" aria-hidden="true" style="margin-right:5px;"></i>
-            <a>Infer Disease Status in Cohorts Lacking Phenotypic Data</a>
-          </li>
-          <div id="inferPanel"
-            style="height: 0px; display: block; overflow: hidden; transition: height 0.5s; -webkit-transition: height 0.5s;">
-            <p>An individual's phenotype can be infered by comparing their risk scores to other individuals in their
-              population or ethnicity.<br>
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12 mb-4 mt-2 text-center">
+          <h2 style="color: #416d9d;">Imputation</h2><br>
+          <p class="text-center">In addition to the above best practices, we also impute SNPs not found in the user's 
+            files. Instead of imputing the reference allele for the population we impute the alleles as unknown. We believe this 
+            to be a more accurate approach as the reference allele is not always the most common allele in the population. By 
+            imputing the alleles as unknown, we are able to use the allele frequency of the risk allele to estimate the 
```

Comment:
Can you clarify that we use the allele frequency of the risk allele in the 1000 genomes cohort? It is not clear that that is what you mean by "allele frequency" here.

Reply:
I made a slight change, I don't know if that is any better.